tools/integration_tests/run_integration_tests.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Again, `integration_tests` is repeated. I would just call it `run.py`.
Note that this only tests pysa, so I could be nice to have it in the name as well. Maybe `tools/pysa/integration_tests` or `tools/pysa_integration_tests`.

Also note that we have integration tests in `stubs/integration_tests`. Does calling this script from there work?
Maybe we should have wrapper scripts then:

`documentation/deliberately_vulnerable_flask_app/run.py`:
```
#!/bin/bash
python3 ../../tools/integration_tests/run.py
```
`stubs/integration_tests/run.py`:
```
#!/bin/bash
python3 ../../tools/integration_tests/run.py
```

Reply:
Is stubs a proper place to have `integration_tests`? I doubt so, CC: @r0rshark what sort of refactoring would you recommend here.

I refractory the files this PR introduces to: `tools/integration_tests/pysa/run.py` and `tools/integration_tests/pysa/utils.py`

Reply:
I like more Maxime suggestion of using `tools/pysa_integration_tests` as a folder. I like having a wrapper like `documentation/deliberately_vulnerable_flask_app/run_integration_tests.py`. 
My remaining concern (not a big one to be honest :D ) is that we may launch the script from a folder which doesn't directly contain the `pyre_configuration` file of the `deliberately_vulnerable_flask_app` e.g. `cd tools/integration_tests && python run.py `. I think this would fail because pyre will check the `pyre_configuration` inside the `tools/integration_tests `. What do you think about using `__file__ ` path to compute where `documentation/deliberately_vulnerable_flask_app/` is and pass it as an argument for specifying the configuration path (`-l`) to the   `run_pysa_integration_test` passthough args

Reply:
> What do you think about using __file__ path to compute where documentation/deliberately_vulnerable_flask_app/ is and pass it as an argument for specifying the configuration path (-l) to the run_pysa_integration_test passthough args

Logical enough, I will make another refraction and get back!